web/public/index.html

```diff
@@ -99,12 +103,39 @@ <h3><i class="material-icons">chat_bubble_outline</i> Friendly Chat</h3>
           </form>
         </div>
       </div>
+    </div>
+    
+    <div hidden id="private-messages-card-container" class="mdl-cell mdl-cell--12-col mdl-grid">
 
+      <!-- Messages container -->
```

Comment:
Maybe move this comment above "private-messages-card-container" and change to "Private messages container"?

Reply:
Let me just say that I'm not convinced we need to duplicate everything here. Doesn't the code below already switch between rooms?